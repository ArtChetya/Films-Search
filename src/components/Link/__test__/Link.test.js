import { shallow } from 'enzyme/build';
import React from 'react';
import { Link } from '..';

describe('Link component', () => {
    it('should be rendered correctly', () => {
        const component = shallow(<Link />);
        expect(component).toMatchSnapshot();
    });
});
