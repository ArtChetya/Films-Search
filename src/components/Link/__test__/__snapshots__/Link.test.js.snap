// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Link component should be rendered correctly 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "Link-sc-1w86vi8-0",
        "isStatic": true,
        "rules": Array [
          "&&{color:initial;text-decoration:none;}",
        ],
      },
      "displayName": "Link",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "Link-sc-1w86vi8-0",
      "target": [Function],
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
